### PR TITLE
elemMods should works as usual

### LINKED ISSUE #156: elemMods should works as usual
I saw https://github.com/bem/bem-react-core/pull/135

Still think that this should works:
```jsx
mix={{
    block: 'foo’,
    elem: 'bar’, 
    elemMods: {mod: 'val’}
}}
```
Because we have some kind of standard of bemjson: https://en.bem.info/platform/bemjson/#conventions-for-standard-bemjson-modes

Thank you!

### PR DESCRIPTION
**Pull request by awinogradov**
Close #156

cc @zxqfox

### CODE REVIEW

**Comment by zxqfox** [comment]
🐰🐰🐰 ❤️

**Comment by awinogradov** [comment]
@Yeti-or no info about mods for elements in docs – I can't change nothing ;)

**Comment by awinogradov** [comment]
I'll add this info in #170

**Comment by awinogradov** [comment]
@Yeti-or I found it! And changes it of course ;)

File: REFERENCE.md [hunk 1]
```diff
@@ -251,19 +251,24 @@ export default decl({
 Field `mix` accepts object or array of objects with next properties:
 
 * block
-* elem
 * mods
+* elem
+* elemMods – back compatibility for the different BEM-tool. `mods` will be ignored in this case.
```
Comment (Yeti-or) [inline REFERENCE.md]: BEM-tools

**Pull request merged**